dali/base/dafdesc.cpp

```diff
@@ -101,7 +101,9 @@ void ClusterPartDiskMapSpec::setRoxie (unsigned redundancy, unsigned channelsPer
 {
     flags = 0;
     replicateOffset = _replicateOffset?_replicateOffset:1;
-    defaultCopies = redundancy+1;
+#ifdef _CONTAINERIZED
```

Comment:
question: Should this still be setting defaultCopies?

Reply:
The defaultCopies default value in containerized mode is now 1, so this change code just leaves it as that.